### DISCUSSION TITLE
Set default language in env variable

### DISCUSSION THREAD

**Discussion by ShlomiD83** [mealie-recipes/mealie#4731] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I would like to set default language in my compose file so the app will always open in Hebrew the first time it opens, but the user can override the setting in Mealie itself. 

### Please provide a concise description of the feature that would resolve your issue.

.. 

### Please consider and list out some caveats or tradeoffs made in your design decision

.. 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
I think this makes sense! I've heard similar requests on Discord before as well
